homeassistant/components/insteon/climate.py

```diff
@@ -90,7 +95,7 @@ def supported_features(self):
         return SUPPORTED_FEATURES
 
     @property
-    def temperature_unit(self) -> str:
+    def temperature_unit(self) -> UnitTemperatureT:
```

Comment:
Please rebase and remove the unrelated commit.

Reply:
Rebased, but I cannot see any unrelated commits. There are 2 commits the original, and the review comments.

Reply:
This commit changes the insteon integration. We should only change the azure devops integration in this PR.

Reply:
Sorry my bad, seems visual code remote did some bad things again. Using visual code remotely (on my iPad) is nice, but it do have problems with committing.

Give me a little time to solve the mess.

Reply:
Again sorry for the confusion, I created the branch from Insteon and not from dev, my fault!

All corrected.